src/natcap/invest/urban_mental_health.py

```diff
@@ -0,0 +1,1335 @@
+"""InVEST Urban Mental Health model."""
+import logging
+import os
+import shutil
+import tempfile
+
+import numpy
+import pandas
+import pygeoprocessing
+import pygeoprocessing.kernels
+from osgeo import gdal
+from osgeo import ogr, osr
+
+from . import gettext
+from . import spec
+from . import validation
+from .unit_registry import u
+
+LOGGER = logging.getLogger(__name__)
+
+SCENARIO_OPTIONS = [
+    spec.Option(key="tcc_ndvi", display_name="TCC & Baseline NDVI"),
+    spec.Option(key="lulc", display_name="Baseline & Alternate LULC"),
+    spec.Option(key="ndvi", display_name="Baseline & Alternate NDVI")
+    ]
+
+MODEL_SPEC = spec.ModelSpec(
+    model_id="urban_mental_health",
+    model_title=gettext("Urban Mental Health"),
+    userguide="",  # TODO - add this model to UG
+    validate_spatial_overlap=True,
+    different_projections_ok=True,
+    aliases=("umh",),
+    module_name=__name__,
+    input_field_order=[
+        ["workspace_dir", "results_suffix"],
+        ["aoi_vector_path", "population_raster", "search_radius"],
+        ["effect_size", "baseline_prevalence_vector",
+         "health_cost_rate"],
+        ["scenario"],
+        ["ndvi_base", "ndvi_alt"],
+        ["lulc_base", "lulc_alt", "lulc_attr_csv"],
+        ["tc_raster", "tc_target"]
+    ],
+    inputs=[
+        spec.WORKSPACE,
+        spec.SUFFIX,
+        spec.N_WORKERS,
+        spec.AOI.model_copy(update=dict(  #  TODO: potentially want to have this req to be census tract pop. shp? or only req that if opt 1 but optional for opts 2-3?
+            id="aoi_vector_path",
```

Comment:
I think in Spec this is defaulted to `aoi_path`. Is there a particular reason not to use this or do you think `aoi_vector_path` would be a better descriptor for what we have in `spec.AOI`, given it's a `VectorInput`?